index.html

```diff
@@ -0,0 +1,33 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <title>Otters</title>
+  <link rel="stylesheet" href="styles/style.css">
+  <link rel="stylesheet" href="styles/index.css">
+</head>
+<body>
+  <header>
+    <h1>About Otters</h1>
+    <nav>
+      <ul>
+        <li><a href="pages/facts.html">Facts</a></li>
+        <li><a href="pages/gallery.html">Gallery</a></li>
+      </ul>
+    </nav>
+    <img id="picture" src="images/HomePageOtter.jpg" width="500" height="600" alt="Otter Image">
```

Comment:
Visually, the page looks good and flows nicely! In my opinion, the img tag might be more appropriate at the top of the main element, above the section element. I say that because a [header],(https://developer.mozilla.org/en-US/docs/Web/HTML/Element/header) per the docs, "represents introductory content, typically a group of introductory or navigational aids."

Since the img isn't a navigational aid, I think it would fit better in main.